src/pages/Card/_utils/formatAmount.ts

```diff
@@ -0,0 +1,4 @@
+// 금액 3자리마다 , 붙이는 함수
+export const formatAmount = (amount: number) => {
+  return amount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+};
```

Comment:
이렇게 util 함수를 구현하셨군요~ 숫자 세자리마다 , 를 붙일 때는 toLocaleString( ) 함수를 사용하는 방법도 제안드리고 싶어요~~ ( 제안이니 참고만 하셔도 됩니다!!)

```suggestion
let num = 123456789;
console.log(num.toLocaleString()); // 123,456,789
```